index.html

```diff
@@ -4,8 +4,22 @@
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
     <title>Les 3 - Opdracht CSS Grid</title>
+
+
 </head>
+
 <body>
+<section class="container">
```

Comment:
Is niet fout maar wellicht was een div logischer geweest als container. Mocht je een CSS style toepassen voor de dit type section neemt hij deze container nu ook mee
```suggestion
<div class="container">
```
Handig dat je zo een suggestie kan geven. Al zal het ook afgesloten moeten worden met div mocht dit worden doorgevoerd.